code/src/components/ToDoList.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import styled from 'styled-components';
+
+import tasks from '../reducers/tasks';
+
+import TaskCard from './TaskCard';
+
+const Container = styled.div`
+width: 100%;
+display; flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+@media (min-width: 768px){
+  width: 680px;
+}
+`;
```

Comment:
remember to separate your container with 1 empty line. It is easier to read 👓

Reply:
Fixed.